tests/e2e/framework/framework.go

```diff
@@ -203,7 +203,7 @@ func (c *CommonConfig) RunTest(m runnable) int {
 		ret = m.Run()
 	}
 	if err := c.saveLogs(ret); err != nil {
-		glog.Warning("Log saving inconplete: %s", err)
+		glog.Warning("Log saving incomplete")
```

Comment:
In latest commit, I retain the err printing, taking the first approach suggested by @ldemailly, because I realize saveLogs() could also encouter errors from not just fetching cluster logs but also creating status file.